getid3/write.id3v2.php

```diff
@@ -966,9 +967,9 @@ public function GenerateID3v2FrameData($frame_name, $source_data_array) {
 					$source_data_array['encodingid'] = (isset($source_data_array['encodingid']) ? $source_data_array['encodingid'] : $this->id3v2_default_encodingid);
 					if (!$this->ID3v2IsValidTextEncoding($source_data_array['encodingid'])) {
 						$this->errors[] = 'Invalid Text Encoding in '.$frame_name.' ('.$source_data_array['encodingid'].')';
-					} elseif (!$this->IsANumber($source_data_array['pricepaid']['value'], false)) {
+					} elseif (!getid3_id3v2::IsANumber($source_data_array['pricepaid']['value'], false)) {
 						$this->errors[] = 'Invalid Price Paid in '.$frame_name.' ('.$source_data_array['pricepaid']['value'].')';
-					} elseif (!$this->IsValidDateStampString($source_data_array['purchasedate'])) {
+					} elseif (!getid3_id3v2::IsValidDateStampString($source_data_array['purchasedate'])) {
```

Comment:
Located in `getid3_id3v2::IsValidDateStampString()`, not in `$this->IsValidDateStampString()`